### ISSUE TITLE
Allow `link[rel=alternate][hreflang=x-default]`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#718]
`hreflang="x-default"` does not conform to the HTML specification, but is used by Google.
https://developers.google.com/search/blog/2013/04/x-default-hreflang-for-international-pages

```html
<link rel="alternate" hreflang="x-default" href="https://www.example.com/" />
```

# Current behavior

Only BCP47 format is allowed for hreflang attribute

# Expected behavior

`hreflang="x-default"` is allowed with preset (like `meta[property]`)

preset.rdfa.json, which allows `meta[property]`:
https://github.com/markuplint/markuplint/blob/712bcbb88823b373fdb4d7b185549bad432b794b/packages/@markuplint/config-presets/src/preset.rdfa.json
